modules/helm/Makefile.repo

```diff
@@ -86,8 +87,8 @@ helm\:repo\:lint:
 helm\:repo\:clean:
 	$(call assert-set,REPO_PATH)
 	@rm -f *.tgz
-	@find ./$(REPO_PATH)/library -maxdepth 3 -mindepth 3 -type f -name '*.tgz' -delete
-	@rm -f ./$(REPO_PATH)/packages/index.yaml ./packages/*.tgz
+	@find ./$(REPO_PATH) -maxdepth 3 -mindepth 3 -type f -name '*.tgz' -delete
+	@rm -f ./packages/$(REPO_PATH)/index.yaml ./packages/*.tgz
```

Comment:
We need to define 3 new ENVs
`REPO_NAME ?= incubator # for example`
`REPO_PATH ?= ./$(REPO_NAME)`
`PACKAGE_PATH ?= ./packages/$(REPO_NAME)`